docs/docsite/rst/user_guide/playbooks_loops.rst

```diff
@@ -26,42 +45,43 @@ To save some typing, repeated tasks can be written in short-hand like so::
          - testuser1
          - testuser2
 
-If you have defined a YAML list in a variables file, or the 'vars' section, you can also do::
+You can define the list in a variables file, or in the 'vars' section of your play, then refer to the name of the list in the task::
 
     loop: "{{ somelist }}"
 
-The above would be the equivalent of::
+Either of these examples would be the equivalent of::
 
     - name: add user testuser1
       user:
         name: "testuser1"
         state: present
         groups: "wheel"
+
     - name: add user testuser2
       user:
         name: "testuser2"
         state: present
         groups: "wheel"
 
-.. note:: Before 2.5 Ansible mainly used the ``with_<lookup>`` keywords to create loops, the `loop` keyword is basically analogous to ``with_list``.
-
-
-Some plugins like, the yum and apt modules can take lists directly to their options, this is more optimal than looping over the task.
-See each action's documentation for details, for now here is an example::
+You can pass a list directly to a parameter for some plugins, like the yum and apt modules. When available, passing the list to a parameter is better than looping over the task. For example::
 
    - name: optimal yum
      yum:
-       name: "{{list_of_packages}}"
+       name: "{{  list_of_packages  }}"
        state: present
 
-   - name: non optimal yum, not only slower but might cause issues with interdependencies
+   - name: non-optimal yum, slower and may cause issues with interdependencies
```

Comment:
I would add note that 'apt' and other package management modules (but not all) have the same features, but that the user should confirm by reading the docs for each module.